core-ui/src/components/BusolaExtensions/BusolaExtensionList.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+import LuigiClient from '@luigi-project/client';
+
+import { ResourcesList } from 'shared/components/ResourcesList/ResourcesList';
+import { Link } from 'shared/components/Link/Link';
+
+import { BusolaExtensionCreate } from './BusolaExtensionCreate';
+
+export function BusolaPluginList(props) {
+  const { t } = useTranslation();
+
+  const customColumns = [
+    {
+      header: t('common.headers.namespace'),
+      value: resource => resource.metadata.namespace,
+    },
+  ];
+
+  const description = (
+    <Trans i18nKey="extensibility.extensions.description">
```

Comment:
We don't have such field in the translations

Reply:
yeah, i was planning on adding it in the next iteration... but i guess i could add something simple now